### ISSUE TITLE
Request for automation of R packages development and submission

### ISSUE THREAD

**Issue by leox31** [Microsoft/RTVS#3757]
Hello,
I am wondering if there are any plans for the development of R packages build and submission. Currently, Creating a package does not seem (at least to myself)  as consistent and difficult to follow as a process. Various authors describe the process differently, and using different development tools, thus making it difficult for data professional like myself to share interesting data sets or functions with other users.
Thanks

**Comment by MikhailArkhipov** [comment]
Yes, see https://github.com/Microsoft/RTVS/issues/1638. No specific date just yet though.
